sys-firmware/ipxe/ipxe-1.21.1_p20230601-r1.ebuild

```diff
@@ -0,0 +1,138 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit savedconfig secureboot toolchain-funcs
+
+# for 1.21.1_p20230601
+COMMIT_SHA1="4fa4052c7ebb59e4d4aa396f1563c89118623ec7"
+
+DESCRIPTION="Open source network boot (PXE) firmware"
+HOMEPAGE="https://ipxe.org/"
+SRC_URI="
+	!binary? ( https://github.com/${PN}/${PN}/archive/${COMMIT_SHA1}.tar.gz -> ${P}.gh.tar.gz )
+	binary? ( https://dev.gentoo.org/~tamiko/distfiles/${P}-bin.tar.xz )"
+S="${WORKDIR}/${PN}-${COMMIT_SHA1}/src"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~loong ~mips ~ppc ppc64 ~riscv ~x86"
```

Comment:
Eek, looks like you missed ppc64 here.

Reply:
**ekeyword ~all your.ebuild** is the surest way to handle keywording on version bumps :)